src/middlewares.js

```diff
@@ -451,6 +467,11 @@ export function promiseEnsureIdempotency(req) {
 }
 
 function invalidRequest(req, res) {
-  res.status(403);
+  res.status(400);
```

Comment:
This should not be changed, 403 is the correct response code for missing authorization.